src/screens/AddEditObservation/AddObservation.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Icon from "react-native-vector-icons/AntDesign";
+import ObservationBase from "./ObservationBase";
+import { translate } from "../../i18n";
+
+const AddObservation = props => {
+  return (
+    <ObservationBase
+      {...props}
+      submitButtonText={translate("addEditObservation.sendObservation")}
+    />
+  );
+};
+AddObservation.navigationOptions = ({ navigation }) => {
```

Comment:
looks like a hack to me 🙃

Reply:
I agree with you, will update to class component

Reply:
FIxed